### DISCUSSION TITLE
Lesson 4: solc version 0.6.0 not install

### DISCUSSION THREAD

**Discussion by syfqabdl** [smartcontractkit/full-blockchain-solidity-course-py#40] [Q&A] [answered]
In lesson 4 [((3:44:35))](https://www.youtube.com/watch?v=M576WGiDBdQ&t=13475s), when I run "deploy.py" I got this error 

> `solcx.exceptions.SolcNotInstalled: solc 0.6.0 has not been installed. Use solcx.install_solc('0.6.0') to install.`

This is the code :

```
from solcx import compile_standard

with open("./simple_storage.sol", "r") as file:
    simple_storage_file = file.read()
    print(simple_storage_file)

# compiled sol

compiled_sol = compile_standard(
    {
        "language": "Solidity",
        "sources": {"simple_storage.sol": {"content": simple_storage_file}},
        "setting": {
            "outputSelection": {
                "*": {"*": ["abi", "metadata", "evm.bytecode", "evm.sourceMap"]}
            }
        },
    },
    solc_version="0.6.0",
)
print(compiled_sol)
```

I have installed py-solc-x 1.1.0

**Comment by AlbertoSinigaglia** [comment, marked as answer]
check this discussion: [https://github.com/smartcontractkit/full-blockchain-solidity-course-py/discussions/5](https://github.com/smartcontractkit/full-blockchain-solidity-course-py/discussions/5)

**Reply by syfqabdl** [reply]
ok got it, thanks Alberto
